common/tasks/linters/yamllint/yaml-lint.yaml

```diff
@@ -15,6 +15,10 @@ spec:
     - name: git-revision
       description: The Git revision (commit SHA) from which the test pipeline is originating.
       type: string
+    - name: config-file
+      description: The configuration file for the YAML linter.
```

Comment:
The path to the YAML linting configuration file located in the Git repository. 
        This configuration file specifies the rules and patterns for linting YAML files.
        It should be relative to the root of the repository.

Reply:
fixed